ctl/import.go

```diff
@@ -46,8 +46,8 @@ type ImportCommand struct {
 	// CreateSchema ensures the schema exists before import
 	CreateSchema bool
 
-	// For Range-Encoded fields, name of the Field to import into.
-	Field string `json:"field"`
+	// For Range-Encoded bsiGroups, name of the BSIGroup to import into.
```

Comment:
again, awk. Just making a note of these, I don't think we need to address them in this PR